src/style.css

```
@@ -0,0 +1,105 @@
+* {
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf) format("truetype");
+  font-weight: 300;
+  font-style: normal;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Heavy.ttf) format("truetype");
+  font-weight: 900;
+  font-style: normal;
+}
+
+html {
+  font-family: Avenir, sans-serif;
+  font-weight: 300;
+}
+
+input,
+input::placeholder {
+  font-family: inherit;
+}
+
+.inputWrapper {
```

it's better to name classes with kebab-case
```suggestion
.input-wrapper {
```

---

I agree with you, but p.6 of Checklist reads another:  6. [CODE STYLE] - Use camelCase for values of name attribute - they should be
   valid as JavaScript object keys. It should not contain spaces, “-” or other
   special characters.  
as per checklist camelCase is preferable.  
Please advise whether it has to be changed on kebab-case or camelCase stays w/o changes?

---

camelCase for values of name
only name attribute)